### PR TITLE
[Doc] change the Chinese proper name "data model" to table type 

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 1]
```diff
@@ -242,5 +242,5 @@ transforms.unwrap.delete.handling.mode
 
 In the above configurations, we specify `transforms=addfield,unwrap`.
 
-- The addfield transform is used to add the __op field to each record of Debezium CDC-formatted data to support the StarRocks primary key model table. If the StarRocks table is not a Primary Key table, you do not need to specify the addfield transform. The addfield transform class is com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord. It is included in the Kafka connector JAR file, so you do not need to manually install it.
+- The addfield transform is used to add the __op field to each record of Debezium CDC-formatted data to support the StarRocks primary key table. If the StarRocks table is not a Primary Key table, you do not need to specify the addfield transform. The addfield transform class is com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord. It is included in the Kafka connector JAR file, so you do not need to manually install it.
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:245]: image
-> Primary Key. See to the other docs involved.

File: docs/en/loading/automq-routine-load.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ For an understanding of the basic principles of Routine Load, refer to the secti
 
 Ensure you have a running StarRocks cluster. For demonstration purposes, this article follow the [deployment guide](../quick_start/deploy_with_docker.md) to install a StarRocks cluster on a Linux machine via Docker.
 
-Creating a database and test table with the primary key model:
+Creating a database and test table with the primary key table:
```
Comment (amber-create) [inline docs/en/loading/automq-routine-load.md:21]: image
two "tables". I think you mean "a Primary Key table for testing". See to this in all docs involved.

File: docs/en/using_starrocks/Using_HLL.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ This is the core idea of the HLL algorithm. Please refer to the [HyperLogLog pap
 ### How to use HyperLogLog
 
 1. To use HyperLogLog de-duplication, you need to set the target indicator column type to `HLL` and the aggregation function to `HLL_UNION` in the table creation statement.
-2. Currently, only the aggregation model supports HLL as indicator column type.
+2. Currently, only the aggregate table supports HLL as indicator column type.
```
Comment (amber-create) [inline docs/en/using_starrocks/Using_HLL.md:62]: image
-> the Aggregate table. Make sure to capitalize all table types (Primary Key, Aggregate, Duplicate Key, and Unique Key) in all docs involved.

File: docs/zh/administration/Profiling.md [hunk 4]
```diff
@@ -152,7 +152,7 @@ StarRocks 支持对数据进行有序存储，在数据有序的基础上为其
 
 ### 使用倒排索引
 
-StarRocks 支持倒排索引，采用位图技术构建索引（Bitmap Index）。您可以将索引应用在 DUPLICATE KEY 数据模型的所有列和 AGGREGATE KEY 和 UNIQUE KEY 模型的 Key 列上。位图索引适合取值空间较小的列，例如性别、城市、省份等信息列上。随着取值空间的增加，位图索引会同步膨胀。
+StarRocks 支持倒排索引，采用位图技术构建索引（Bitmap Index）。您可以将索引应用在明细表的所有列，聚合表和更新表的 Key 列上。位图索引适合取值空间较小的列，例如性别、城市、省份等信息列上。随着取值空间的增加，位图索引会同步膨胀。
```
Comment (amber-create) [inline docs/zh/administration/Profiling.md:155]: image
名词并列用顿号

File: docs/zh/loading/Flink-connector-starrocks.md [hunk 5]
```diff
@@ -110,7 +110,7 @@ Flink connector JAR 文件的命名格式如下：
 | sink.max-retries                  | No       | 3             | Stream Load 失败后的重试次数。超过该数量上限，则数据导入任务报错。取值范围：[0, 10]。该参数只在 `sink.version` 为 `V1` 才会生效。 |
 | sink.connect.timeout-ms           | No       |  30000            | 与 FE 建立 HTTP 连接的超时时间。取值范围：[100, 60000]。  Flink connector v1.2.9 之前，默认值为 `1000`。  |
 | sink.wait-for-continue.timeout-ms | No       | 10000         | 此参数自 Flink connector 1.2.7 开始支持。等待 FE HTTP 100-continue 应答的超时时间。取值范围：[3000, 60000]。 |
-| sink.ignore.update-before         | No       | TRUE          | 此参数自 Flink connector 1.2.8 开始支持。将数据导入到主键模型表时，是否忽略来自 Flink 的 UPDATE_BEFORE 记录。如果将此参数设置为 false，则将该记录在主键模型表中视为DELETE 操作。 |
+| sink.ignore.update-before         | No       | TRUE          | 此参数自 Flink connector 1.2.8 开始支持。将数据导入到主键表时，是否忽略来自 Flink 的 UPDATE_BEFORE 记录。如果将此参数设置为 false，则将该记录在主键表中视为DELETE 操作。 |
```
Comment (amber-create) [inline docs/zh/loading/Flink-connector-starrocks.md:113]: image
加个空格

File: docs/zh/loading/Flink-connector-starrocks.md [hunk 6]
```diff
@@ -209,7 +209,7 @@ Flink connector 提供以下指标来监控导入情况。
 
 #### 创建 StarRocks 表
 
-创建数据库 `test`，并创建主键模型表  `score_board`。
+创建数据库 `test`，并创建主键表  `score_board`。
```
Comment (amber-create) [inline docs/zh/loading/Flink-connector-starrocks.md:212]: image
多余空格

File: docs/zh/release_notes/release-1.19.md [hunk 7]
```diff
@@ -12,7 +12,7 @@ displayed_sidebar: "Chinese"
 
 * 实现Global Runtime Filter，可以支持对shuffle join实现Runtime filter。
 * 默认开启CBO Planner，完善了colocate join / bucket shuffle / 统计信息等功能。[参考文档](https://docs.starrocks.io/zh/docs/using_starrocks/Cost_based_optimizer/)
-* [实验功能] 发布主键模型（Primary Key）：为更好地支持实时/频繁更新功能，StarRocks新增了一种表的类型: 主键模型。该模型支持Stream Load、Broker Load、Routine Load，同时提供了基于Flink-cdc的MySQL数据的秒级同步工具。[参考文档](https://docs.starrocks.io/zh/docs/table_design/table_types/primary_key_table/)
+* [实验功能] 发布主键表（Primary Key）：为更好地支持实时/频繁更新功能，StarRocks新增了一种表的类型: 主键表。该类型的表支持Stream Load、Broker Load、Routine Load，同时提供了基于Flink-cdc的MySQL数据的秒级同步工具。[参考文档](https://docs.starrocks.io/zh/docs/table_design/table_types/primary_key_table/)
```
Comment (amber-create) [inline docs/zh/release_notes/release-1.19.md:15]: image
加个空格

File: docs/zh/release_notes/release-2.0.md [hunk 8]
```diff
@@ -147,7 +147,7 @@ displayed_sidebar: "Chinese"
 - Broker Load支持华为OBS [#1182](https://github.com/StarRocks/starrocks/pull/1182)
 - 支持国密算法sm3
 - 适配ARM类国产CPU：通过鲲鹏架构验证
-- 主键模型（Primary Key）正式发布，该模型支持Stream Load、Broker Load、Routine Load，同时提供了基于Flink-cdc的MySQL数据的秒级同步工具。[参考文档](https://docs.starrocks.io/zh/docs/table_design/table_types/primary_key_table/)
+- 主键表正式发布，该类型的表支持Stream Load、Broker Load、Routine Load，同时提供了基于Flink-cdc的MySQL数据的秒级同步工具。[参考文档](https://docs.starrocks.io/zh/docs/table_design/table_types/primary_key_table/)
```
Comment (amber-create) [inline docs/zh/release_notes/release-2.0.md:150]: image
空格

File: docs/zh/release_notes/release-2.3.md [hunk 9]
```diff
@@ -241,7 +241,7 @@ displayed_sidebar: "Chinese"
 - BE 负载很高时，资源组的监控指标 starrocks_be_resource_group_running_queries 统计错误。[#14043](https://github.com/StarRocks/starrocks/pull/14043)
 - 如果查询语句使用 OUTER JOIN，可能会导致 BE 节点崩溃。[#14840](https://github.com/StarRocks/starrocks/pull/14840)
 - 2.4 版本使用了异步物化视图后回滚到 2.3 版本，导致 FE 无法启动。[#14400](https://github.com/StarRocks/starrocks/pull/14400)
-- 主键模型的表部分地方使用了delete_range，如果性能不佳可能会导致后续从 RocksDB 读取数据过慢，导致 CPU 资源占用率过高。[#15130](https://github.com/StarRocks/starrocks/pull/15130)
+- 主键表部分地方使用了delete_range，如果性能不佳可能会导致后续从 RocksDB 读取数据过慢，导致 CPU 资源占用率过高。[#15130](https://github.com/StarRocks/starrocks/pull/15130)
```
Comment (amber-create) [inline docs/zh/release_notes/release-2.3.md:244]: image
空格，以及加 code format

File: docs/zh/table_design/Table_design.md [hunk 10]
```diff
@@ -4,4 +4,4 @@ displayed_sidebar: "Chinese"
 
 # 介绍
 
-本章首先介绍StarRocks的数据模型，用户可根据业务需要选择匹配的模型; 然后在这个基础上，介绍数据分布策略; 最后介绍索引技术和物化视图处理技术。
+本章首先介绍StarRocks的表类型，用户可根据业务需要选择匹配的表类型; 然后在这个基础上，介绍数据分布策略; 最后介绍索引技术和物化视图处理技术。
```
Comment (amber-create) [inline docs/zh/table_design/Table_design.md:7]: image
空格

File: docs/zh/table_design/table_types/table_types.md [hunk 11]
```diff
@@ -2,26 +2,26 @@
...
-建表时，您需要指定数据模型 (Data Model)，这样数据导入至数据模型时，StarRocks 会按照排序键对数据进行排序、处理和存储。本文介绍 StarRocks 支持的各种数据模型，满足您在不同业务场景下的需求。
+建表时，您需要指定表类型 (Table Type)，这样数据导入至表时，StarRocks 会按照排序键对数据进行排序、处理和存储。本文介绍 StarRocks 支持的各种表类型，满足您在不同业务场景下的需求。
 
 ## 基本概念
 
-### 数据模型
+### 表类型
 
-StarRocks 支持四种数据模型，分别是明细模型 (Duplicate Key Model)、聚合模型 (Aggregate Key Model)、更新模型 (Unique Key Model) 和主键模型 (Primary Key Model)。这四种数据模型能够支持多种数据分析场景，例如日志分析、数据汇总分析、实时分析等。
+StarRocks 支持四种表类型，分别是明细表 (Duplicate key table)、聚合表 (Aggregate table)、更新表 (Unique key table) 和主键表 (Primary key table)。这四种表类型能够支持多种数据分析场景，例如日志分析、数据汇总分析、实时分析等。
```
Comment (amber-create) [inline docs/zh/table_design/table_types/table_types.md:13]: image
表类型的大小写，同英文，需要全局关注

File: docs/zh/using_starrocks/Using_HLL.md [hunk 12]
```diff
@@ -10,7 +10,7 @@ HLL 是一种近似去重算法，在部分对去重精度要求不高的场景
 
 ## 创建包含 HLL 列的表
 
-使用 HLL 去重，需要在建表语句中，将目标指标列的类型设置为 **HLL**，聚合函数设置为 **HLL_UNION**。只有聚合模型表 (Aggregate Key) 支持 HLL 类型列。
+使用 HLL 去重，需要在建表语句中，将目标指标列的类型设置为 **HLL**，聚合函数设置为 **HLL_UNION**。只有聚合表 (Aggregate Key) 支持 HLL 类型列。
```
Comment (amber-create) [inline docs/zh/using_starrocks/Using_HLL.md:13]: image
中英文全局排查“聚合表”的写法，应该是“Aggregate table”，不加 Key

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**